fgcm/fgcmStars.py

```diff
@@ -1167,13 +1167,11 @@ def plotStarMap(self,mapType='initial'):
             # Requested not to do plots
             return
 
-        import healpy as hp
-        try:
-            from .fgcmPlotmaps import plot_hpxmap
-        except ImportError:
-            self.fgcmLog.info("Map plotting not available.  Sorry!")
-            return
+        # This is not currently used.
+        # FIXME: add skyproj plotting.
+        return
 
+    """
```

Comment:
I don't think disabling code like this is complying with LSST code standards.

Reply:
Probably not, but I wanted to keep it around to when I reenable it using `skyproj`.